tests/api-controller-acceptance-tests/apicontroller/apicontroller_test.go

```diff
@@ -135,7 +136,87 @@ func TestSpec(t *testing.T) {
 			So(lastApi.ResourceVersion, ShouldNotBeEmpty)
 		})
 
-		Convey("validate updated API", func() {
+		Convey("validate updated API with default jwt configuration", func() {
+
+			t.Log("Validate updated...")
+
+			checkPreconditions(lastApi, t)
+			validateApiSecured(httpClient, lastApi)
+		})
+
+		Convey("get updated API with default jwt configuration", func() {
+
+			t.Log("Get...")
+
+			gotApi, err := kymaInterface.GatewayV1alpha2().Apis(namespace).Get(lastApi.Name, metav1.GetOptions{})
+
+			So(err, ShouldBeNil)
+			So(gotApi, ShouldNotBeNil)
+
+			lastApi = gotApi
+
+			So(lastApi.ResourceVersion, ShouldNotBeEmpty)
+		})
+
+		Convey("update API to disable authentication", func() {
+
+			t.Log("Update...")
+
+			api := apiFor(testId, domainName, fixture.SampleAppService, apiSecurityDisabled)
+			api.ResourceVersion = lastApi.ResourceVersion
+
+			updatedApi, err := kymaInterface.GatewayV1alpha2().Apis(namespace).Update(api)
+
+			So(err, ShouldBeNil)
+			So(updatedApi, ShouldNotBeNil)
+
+			lastApi = updatedApi
+
+			So(lastApi.ResourceVersion, ShouldNotBeEmpty)
+		})
+
+		Convey("validate updated API with disabled authentication", func() {
```

Comment:
not needed here, make such assertion in above scenario

Reply:
done